src/query.jl

```diff
@@ -172,3 +172,50 @@ function get_for_cids(cids;
 end
 
 get_for_cids(cid::Int; kwargs...) = get_for_cids([cid]; kwargs...)
+
+"""
+   `pug(args...; silent = false, escape_args = true, return_text = true, status_exception = false, kwargs...)`
+
+Generate a PUG endpoint and call it. The details about PUG endpoints are described here: <https://pubchem.ncbi.nlm.nih.gov/docs/pug-rest>
+
+Keyword arguments:
+
+- `escape_args = true`, URL encodes each argument before generating the endpoint.
+Setting this false is useful when copy-pasting an existing PUG endpoint, e.g. from documentation.
+
+- `silent = true` will suppress printing the pug URL called.
+
+
+This function uses more ergonomic defaults that the rest of the package:
```

Comment:
What about aiming for consistency here and then having a breaking release where we change them all?